charts/studio/values.yaml

```diff
@@ -238,6 +238,21 @@ postgresql:
         # -- Postgres database
         database: iterativeai
 
+clickhouse:
+  # -- ClickHouse enabled
+  enabled: false
+  # -- ClickHouse name override
+  fullnameOverride: studio-clickhouse
+
+  # Shards / replicas configuration
+  replicaCount: 1
+  shards: 1
```

Comment:
Using more shards/replicas will cause eventual consistency issues with table creation; to be fiexed whenever we have to scale.